### PR TITLE
Add vnodes default configuration file.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

SSIA

##### Test Plan

n/a

##### Additional Information

Closes: #14546

Requires https://github.com/netdata/netdata/pull/14545 to be merged before this is fully usable.

### CODE REVIEW

**Review by tkatsoulas — APPROVED** [review]

**Comment by ilyam8** [comment]
@Ferroin renamed the file to `.conf` because all ND config files have `.conf` suffix.

**Comment by ilyam8** [comment]
@Ferroin I get `vnodes.conf` in libconfig, not in libconfig/vnodes/

```cmd
$ ls -l /opt/netdata/usr/lib/netdata/conf.d/ | grep vnodes
.rw-r--r--  854 root 10 Mar 19:08 vnodes.conf
```

File: system/vnodes/vnodes.conf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+##
+## You can create GUIDs online: https://www.guidgen.com/
+## or generate with the linux command: uuidgen
+##
+## ---------------------------------------------------------------------------------------------------------------------
+
+#- hostname: HOSTNAME1
+#  guid: GUID1
+#
+#- hostname: HOSTNAME1
```
Comment (ilyam8) [inline system/vnodes/vnodes.conf:22]:
```suggestion
#- hostname: HOSTNAME2
```

**Comment by Ferroin** [comment]
> @Ferroin renamed the file to `.conf` because all ND config files have `.conf` suffix.

We should really get away from this for our YAML config files. I can understand the not-quite-TOML stuff like netdata.conf keeping the `.conf` extension, but for anything that accepts a YAML file, it should use the correct file extension for the file format to avoid confusion (especially if we’re moving all of our config to YAML).

**Comment by Ferroin** [comment]
> @Ferroin I get `vnodes.conf` in libconfig, not in libconfig/vnodes/
> 
> ```batchfile
> $ ls -l /opt/netdata/usr/lib/netdata/conf.d/ | grep vnodes
> .rw-r--r--  854 root 10 Mar 19:08 vnodes.conf
> ```
> 
> Which is probably fine. Perhaps we need to add `vnodes/` dir only if there will be a need and keep it simple for now. What do you think?

Should be fixed now, it the intent was to have a vnodes directory.

File: system/Makefile.am [hunk 2]
```diff
@@ -26,6 +26,7 @@ dist_config_DATA = \
     .install-type \
     $(NULL)
 
+libconfigvnodesdir=$(libconfig)/vnodes
```
Comment (ilyam8) [inline system/Makefile.am:29]:
```suggestion
libconfigvnodesdir=$(libconfigdir)/vnodes
```

**Comment by ilyam8** [comment]
@Ferroin lgtm now, please fix merge conflicts.

**Comment by Ferroin** [comment]
Rebased to resolve merge conflicts.

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Pull request merged**